src/btree/bt_vrfy_dsk.c

```diff
@@ -293,7 +293,7 @@ __verify_dsk_row(
 		case WT_CELL_ADDR_LEAF_NO:
 		case WT_CELL_KEY_OVFL:
 		case WT_CELL_VALUE_OVFL:
-			if (!bm->addr_valid(bm,
+			if (bm->addr_invalid(bm,
 			    session, unpack->data, unpack->size))
```

Comment:
Do we need to distinguish between `EINVAL` and other errors here (e.g., `ENOMEM`, `WT_PANIC`)?